### PR TITLE
Add contributions and about pages

### PR DESCRIPTION
**Pull request by amlmn**

### CODE REVIEW

File: html/contributions.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
+<head>
+  <meta charset="utf-8" />
+  <meta name="generator" content="pandoc" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
+  <title>Contributions</title>
+  <style type="text/css">
```
Comment (amlmn) [inline html/contributions.html]: @wbthomason, do you know the magic pandoc flag to suppress this css?

**Comment by wbthomason** [comment]
This all seems generally fine (I have a one-word tweak to the "About" text to propose), and I do know how to fix the Pandoc issue. However, the fix involves using a custom template file I've made (really just adapting the Pandoc default to use our styles, etc.), and it strikes me that this + the raw .md files + the Python for building the site really shouldn't be on this branch; they should be on a `source` branch or similar. Thoughts, @amlmn?

**Comment by amlmn** [comment]
> 
> 
> This all seems generally fine (I have a one-word tweak to the "About" text to propose), and I do know how to fix the Pandoc issue. However, the fix involves using a custom template file I've made (really just adapting the Pandoc default to use our styles, etc.), and it strikes me that this + the raw .md files + the Python for building the site really shouldn't be on this branch; they should be on a `source` branch or similar. Thoughts, @amlmn?

What's the alternative to having it in master? Github will only serve from master for `<orgname>.github.io` domains.

**Comment by wbthomason** [comment]
The alternative would be: 
- Host HTML on `master`
- Use `master` as the `<orgname>.github.io` domain branch
- Host raw Markdown/templates/the build script on a `source` branch

We would need to set up the `source` branch by doing the following once. Assume that the results of running the build script, i.e. the generated HTML, are placed in a directory `build/`:
- `echo "build/" >> .gitignore`
- `git worktree add build master`

Then building/deploying the site becomes: 
**On `source`**:
- Run the build script. The results are placed in `build/`.
- Inside `build/`:
  - `git add --all`
  - `git commit -m "Deploy article on Foo"`
  - `git push origin master`

And all that could be built into the build tool/a Makefile/whatever.

Reference: https://gist.github.com/cobyism/4730490#gistcomment-2375522

**Comment by wbthomason** [comment]
This is *slightly* more of a pain than just doing everything on `master`, but it's cleaner in terms of separating build results and build source from what's available through the website.

**Review by wbthomason — COMMENTED** [review]
Review is pending discussion in main comments.

**Comment by wbthomason** [comment]
Out-of-band discussion determined that the source branch solution was acceptable.

@amlmn With your OK, I will:
- Close this PR without merging
- Create a new PR for the new HTML files
- Create a new PR to delete the files which are now on `source` only (the Python and raw Markdown)

I've already created a `source` branch to use for this purpose.

**Comment by amlmn** [comment]
SGTM

**Comment by wbthomason** [comment]
:+1: Closing.

**Pull request closed without merging**